.gitignore

```diff
@@ -0,0 +1 @@
+papers/.DS_Store
```

Comment:
Do you want me to just manually remove this after the pull? Is it too hard to get rid of?

Reply:
I think that's just the git ignore entry. I can have git ignore ignore itself.

Reply:
LOL, I'm an idiot. Thanks for bearing with me.